relay/nakama/handlers.go

```diff
@@ -0,0 +1,255 @@
+package main
+
+import (
+	"context"
+	"database/sql"
+	"encoding/json"
+	errors2 "errors"
+	"github.com/heroiclabs/nakama-common/runtime"
+	"github.com/rotisserie/eris"
+	"io"
+	"net/http"
+	"pkg.world.dev/world-engine/relay/nakama/allowlist"
+	"pkg.world.dev/world-engine/relay/nakama/errors"
+	"pkg.world.dev/world-engine/relay/nakama/persona"
+	"pkg.world.dev/world-engine/relay/nakama/receipt"
+	"pkg.world.dev/world-engine/relay/nakama/utils"
+	"strings"
+)
+
+// handleClaimPersona handles a request to Nakama to associate the current user with the persona tag in the payload.
+func handleClaimPersona(verifier *persona.Verifier, notifier *receipt.Notifier) nakamaRPCHandler {
+	return func(
+		ctx context.Context,
+		logger runtime.Logger,
+		db *sql.DB,
+		nk runtime.NakamaModule,
+		payload string,
+	) (string, error) {
+		ptr := &persona.StorageObj{}
+		if err := json.Unmarshal([]byte(payload), ptr); err != nil {
+			return utils.LogErrorWithMessageAndCode(
```

Comment:
We have a lot of different utils.Log... helpers. I think they should be consolidated into 1 or 2 helpers that are easier to use. This is for another PR though. Here's the tracking ticket: https://linear.app/arguslabs/issue/WORLD-784/review-utilslog-helpers